### DISCUSSION TITLE
[Support] how to undo dhcp reservation

### DISCUSSION THREAD

**Discussion by alec-jensen** [pivpn/pivpn#1479] [Support]
## In raising this issue, I confirm the following:

`{please fill the checkboxes, e.g: [X]}`

- [X] I have read the [documentation](https://docs.pivpn.io)
- [X] I have read and understood the [contributors guide](https://github.com/pivpn/pivpn/blob/master/CONTRIBUTING.md).
- [x] The issue I am reporting can be *replicated*.
- [X] The issue I am reporting can be *is* directly related to the pivpn installer script.

### Have you searched for similar issues and solutions?
    (yes/no / which issues?)
    yes how to undo dhcp reservation
### Has your install failed? 
```
yes/no
```

#### What type of hardware are you running PiVPN at?

```
Raspberrypi 3b
```
#### Output of `uname -a`
```
i cant put it, i cant ssh to my pi to copy it
```

#### Output of `cat /etc/os-release`
```
i cant put it, i cant ssh to my pi to copy it
```

### If install failed Please provide the console output of  `curl -L https://install.pivpn.io | bash`
```
  OUTPUT HERE / ANSWER HERE
  DO NOT DELETE THE BACK-TICKS 
  PASTE THE OUTPUT INSIDE THE BACK-TICKS
```

### Issue

how can i undo the dhcp reservation? i set it up on my router, and need to undo it on my pi. i need to undo it on my pi because it cant get an ip

### Console output of      `curl -L install.pivpn.io | bash`
```
i cant put it, i cant ssh to my pi to copy it
```

### Console output of      `pivpn add` or `pivpn add nopass`
```
i cant put it, i cant ssh to my pi to copy it
```

### Console output of      `pivpn debug`

```
i cant put it, i cant ssh to my pi to copy it
```
### Have you taken any steps towards solving your issue?
```
  i googled it
```

**Comment by coolapso** [comment]
Moved to discussions. 

Please be more clear with what you are asking. 

If i recall correctly (haven't touch pivpn code in a while) ... DHCP reservation the setup doesn't touch your IP configuration and let's your router to take care of it. 
Check your router for what IP it has been assigned to your raspberry pi. 
nothing more we can do to assist.

**Comment by orazioedoardo** [comment]
Open `/etc/dhcpcd.conf` and remove this section at the bottom.
```
interface eth0
        static ip_address=192.168.1.45/24
        static routers=192.168.1.1
        static domain_name_servers=127.0.0.1
```
